Apps/WebClient/src/featuretoggleconfig.json

```diff
@@ -89,5 +89,8 @@
                 "enabled": true
             }
         ]
+    },
+    "sharing": {
+        "enabled": true
     }
```

Comment:
this is a little unclear its added in a future task, but it could already be added here to behonest, otherwise we'll need to update this component again later when we introduce filtering.... I'll leave this unresolved for the weekend and make a decision on Monday if I should add it here and remove the future requirement in the next task

Reply:
I've decided to kick this down the road, I've got a bad flu today and this needs to be in to keep the feature development on track, an additional consideration is:

1. Our common convention is to use "enabled" wording, where the default value is disabled.
2. However, this situation is in reversed, our default is enabled and to turn it off would be "delegationEnabled : false" which seems counterintuitive.

I would argue a better implementation is to use "delegationDisabled" which is better aligned to the default behaviour.

I'm putting a link to this conversation in the future task.